apps/condo/domains/news/components/NewsForm/InputStep/InputStepSelector.tsx

```diff
@@ -225,8 +225,15 @@ export const InputStepSelector: React.FC<InputStepSelectorProps> = ({
         showArrow:false,
         infinityScroll:true,
         placeholder:CustomSelectRecipientsPlaceholder,
-        onChange:(recipients: GetNewsSharingRecipientsQuery['recipients']) => {
-            setSharingAppFormValues(prev=>({ ...prev, scope: recipients, isAllChecked: false }))
+        onChange:(recipients: Array<string>) => {
+            try {
+                const parsedRecipients = recipients.map(recipient => JSON.parse(recipient)?.value)
```

Comment:
Maybe be better: 
```suggestion
                const parsedRecipients = recipients.map(recipient => recipient && JSON.parse(recipient)?.value)
```

In this case you don't need try-catch or not?

Reply:
if the recipient cannot parse to json, then there will be error, so this check will not help